src/sentry/tasks/post_process.py

```diff
@@ -489,7 +489,8 @@ def run_post_process_job(job: PostProcessJob):
 
     for pipeline_step in pipeline:
         try:
-            pipeline_step(job)
+            with sentry_sdk.start_span(op=f"tasks.post_process_group.{pipeline_step}"):
```

Comment:
The `pipeline_step` is a function (or callable), so templating the name like this will not produce a stable name for the pipeline step (maybe even invalid).

```
>>> def f():
...    pass
...
>>> f"{f}"
'<function f at 0x107e0f700>'
```

To get the function name use `pipeline_step.__name__`.